test/performance/modules/PCI-DSS-module-dashboard.js

```diff
@@ -0,0 +1,48 @@
+const { WAIT_TIMEOUT, SERVER_URL } = require('../common/constants');
+const logger = require('../common/logger');
+
+module.exports = async function (context, commands) {
+  // Navigate to a URL, but do not measure the URL
+  await commands.navigate(SERVER_URL);
+
+  try {
+    // Click on Kibana menu to access Wazuh App link
+    await commands.wait.bySelector('button[data-test-subj="toggleNavButton"]', WAIT_TIMEOUT)
+    await commands.click.bySelector('button[data-test-subj="toggleNavButton"]')
+    await commands.wait.bySelector('a[href$="/app/wazuh"]', WAIT_TIMEOUT)
+    await commands.click.bySelector('a[href$="/app/wazuh"]')
+    
+    //Wait for an Wazuh home page component to be loaded
+    await commands.wait.byXpath('//*[contains(@class,"euiTitle euiTitle--small euiCard__title")]//*[contains(text(),"PCI DSS")]', WAIT_TIMEOUT)
+    await commands.wait.byCondition("!isNaN(parseInt(document.querySelector('.statWithLink').innerHTML))", WAIT_TIMEOUT)
+    await commands.click.byXpath('//*[contains(@class,"euiTitle euiTitle--small euiCard__title")]//*[contains(text(),"PCI DSS")]')
+
+    // Start collecting metrics
```

Comment:
Metrics must start between lines 17 and 18.
The test ignores some requests if the metrics start past that point
![Screenshot from 2022-08-04 07-53-54](https://user-images.githubusercontent.com/104914131/182831661-15340946-27db-4328-a21a-89884c1a7db9.png)
![Screenshot from 2022-08-04 07-54-32](https://user-images.githubusercontent.com/104914131/182831837-d99de457-d162-4db8-8576-5e67aeac7145.png)

Reply:
Done